### PR TITLE
Update UI to match new dashboard style

### PR DESCRIPTION
**Pull request by lkostrowski**
Updated for few apps - some of them failed with theme provider

Merge this when dashboard is merged

### CODE REVIEW

File: apps/klaviyo/pages/_app.tsx [hunk 1]
```diff
@@ -24,14 +53,19 @@ const themeOverrides: Partial<Theme> = {
   },
 };
 
+const generateClassName = createGenerateClassName({
+  productionPrefix: "c",
```
Comment (krzysztofwolski) [inline apps/klaviyo/pages/_app.tsx:57]: What `c` stands for? :thinking:

File: apps/search/src/pages/_app.tsx [hunk 2]
```diff
@@ -30,12 +35,36 @@ const queryClient = new QueryClient({
...
+      default: "hsla(211, 42%, 14%, 1)",
+      paper: "hsla(211, 42%, 14%, 1)",
+    },
+  },
+};
 
 /**
  * That's a hack required by Macaw-UI incompatibility with React@18
+ *
+ * todo use from packages/shared
```
Comment (krzysztofwolski) [inline apps/search/src/pages/_app.tsx:64]:
```suggestion
 * TODO: use from packages/shared
```

It's a nitpick, but with such formatting y editor automatically highlights the line :)

File: .changeset/shy-gorillas-shop.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"@saleor/apps-shared": minor
```
Comment (krzysztofwolski) [inline .changeset/shy-gorillas-shop.md:2]: Apps depending on shared module probably should be added in this note too.

**Review by krzysztofwolski — CHANGES_REQUESTED** [review]

File: apps/klaviyo/pages/_app.tsx (hunk 1 above)
Comment (lkostrowski) [reply apps/klaviyo/pages/_app.tsx:57]: It was copy pasted from stack overflow 😆 

Bascially its some workaround to material/jss that scopes generated styles so they don't conflict. 
Without that, code from `packages/shared` was duplicating classnames

**Review by krzysztofwolski — APPROVED** [review]

**Pull request merged**